extensions/wikia/MercuryApi/handlers/MercuryApiMainPageHandler.class.php

```diff
@@ -39,21 +45,20 @@ public static function getMainPageData( MercuryApi $mercuryApiModel ) {
 		return $mainPageData;
 	}
 
-	public static function getCuratedContentData( MercuryApi $mercuryApiModel, $section = null ) {
+	public static function getCuratedContentData( MercuryApi $mercuryApiModel, $section = null, $newFormat=false ) {
 		$data = [ ];
-
 		try {
 			$data = WikiaDataAccess::cache(
-				self::curatedContentDataMemcKey( $section ),
+				self::curatedContentDataMemcKey( $section . $newFormat ),
```

Comment:
Before I added it, I had caching issues that caused old format to be put within the 'curatedMainPageData'. After release this should return to previous state